components/AntHiveBotHorizontal.vue

```diff
@@ -83,19 +125,70 @@ export default {
 
   .name {
     font-size: $font-big;
+    margin-bottom: 12px;
+    .display-name {
+      margin-right: 6px;
+      font-weight: $font-weight-bold;
+    }
+    @media (min-width: $screen-md) and (max-width: $screen-lg) {
+      margin-bottom: 0;
+    }
   }
 
   .description {
-    position: relative;
-    padding: 20px 13px;
+    padding: 14px 20px 14px 20px;
+    width: 100%;
+    @media (min-width: $screen-md) and (max-width: $screen-lg) {
+      padding: 10px;
+    }
   }
 
   .statistics {
-    margin-top: 12px;
     display: flex;
-    p {
-      margin-bottom: 0;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    .statistic-container {
+      width: 80px;
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between;
+    }
+    .statistic {
+      color: $color-violet-700;
+      margin-bottom: 5px;
       font-size: $font-medium;
+      display: flex;
+      justify-content: space-between;
+      @media (min-width: $screen-md) and (max-width: $screen-lg) {
+        margin-bottom: 0;
+      }
+    }
+    .statistic-value {
+      display: block;
+      font-weight: $font-weight-bold;
+    }
+  }
+  .information {
+    position: absolute;
+    min-width: 100%;
+    height: 100%;
+    transition: all 0.4s;
+    &:hover {
+      background: #040505b8;
```

Comment:
remove to global, we will use it for enother components too

Reply:
or you can use `$color-black-transparent: #000000d1;` from global.scss
I added it in my last PR for hover layouts